src/components/overview/overview.module.css

```diff
@@ -175,4 +175,76 @@
   width: 60px;
 
 
+}
+
+
+
+
+/*============= Image Gallery ====================*/
+
+.galleryContainer {
+  display: flex;
+  flex-direction: row;
+  height: 100%;
+  width: 100%;
+  border: 1px solid black;
+
+
+}
+
+.galleryThumbnails{
+  height: 680px;
+  width: 100px;
+  margin: 10px 2.5px;
+  border: 1px solid purple;
+
+  display: flex;
+  flex-direction: column;
+}
+
+
+.galleryThumbnailPhotoContainer{
+  display: flex;
+  height: 75px;
+  width: 75px;
+  margin: 8px;
+  /* border: 2px solid black; */
+  justify-content: center;
+  /* align-items: center; */
+  /* vertical-align: middle; */
+
+}
+
+.galleryThumbnailPhoto{
+  height: 100%;
+  width: auto;
+  /* padding: .75px;
+  border: 2px solid black; */
+  border-radius: 5%;
+}
+
+.galleryThumbnailPhotoSelected{
+  height: 100%;
+  width: auto;
+  padding: 1px;
+  border: 2px solid black;
+  border-radius: 5%;
+}
+
+
+.galleryThumbnailPhoto:hover {
+  opacity: .4;
+}
+
+.galleryMain{
+  height: 680px;
+  width: 550px;
+  margin: 10px;
+  border: 1px solid purple;
+}
+
+.mainPhoto{
+  height: 100%;
+  width: auto;
+  justify-content: ;
```

Comment:
Is this not supposed to have a value assigned?

Reply:
Removed extra unassigned property. Was used for testing flexbox.